External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -2667,20 +2673,15 @@ private static ASResult EvalExpression(string expression, ASExpr context, FileMo
                     // eval sub expression
                     head = EvalExpression(subExpr, subContext, inFile, inClass, true, false);
                     if (head.Member != null)
-                        head.Type = ASContext.Context.ResolveType(head.Member.Type, head.Type.InFile);
+                        head.Type = ctx.ResolveType(head.Member.Type, head.Type.InFile);
                 }
                 else
                 {
                     token = token.Substring(token.IndexOf('~') + 1);
                     head = EvalVariable(token, context, inFile, inClass);
                 }
             }
-            else if (token == "\"") // literal string
-            {
-                head = new ASResult();
-                head.Type = ASContext.Context.ResolveType(ASContext.Context.Features.stringKey, null);
-            }
-            else if (token.Contains("<")) head = new ASResult {Type = ASContext.Context.ResolveType(token, inFile)};
+            else if (token.Contains("<")) head = new ASResult {Type = ctx.ResolveType(token, inFile)};
```

Comment:
You added some lines above `else if (features.hasE4X && token == "</>") ` so it looks like it will be overriden by this line?